samples/dotnet_core_managed_gateway/README.md

```diff
@@ -0,0 +1,102 @@
+.NET Core Managed Gateway Sample
+================================
+
+Overview
+--------
+
+This sample showcases how one might build an IoT Gateway as a .NET Core Application and still be able to load .NET Core Modules. 
+
+The sample contains:
+
+1. A printer module (C#) that interprets telemetry from sensor and prints its content and properties into Console.
+2. A sensor module (C#) that publishes random data to the gateway.
+3. A logger module for producing message broker diagnostics.
+
+Other resources:
+- Hello World [sample](../hello_world/README.md)
+- .NET Core Sample [dotnetcoresample] (../dotnet_core_module_sample/README.md)
+- [Devbox setup](../../doc/devbox_setup.md)
+- [.NET Core binding High Level Design](../../bindings/dotnetcore/devdoc/dotnet_core_binding_hld.md)
```

Comment:
I don't think we want to mention anything from the `devdoc/` directory in a sample's README. The HLD is an internal technical document. People are free to read it because it's there, but it isn't part of our public documentation.